client/src/plus/index.scss

```diff
@@ -34,7 +39,7 @@
       height: 100%;
       z-index: 10000;
       overflow: auto;
-      background-color: #201139c4;
+      background-color: #291f35c4;
       -webkit-box-sizing: border-box;
       box-sizing: border-box;
```

Comment:
I do not think we need this as it is set globally here:
https://github.com/mdn/mdn-minimalist/blob/0ec61f3e48a2539297e54b2f7363df5b76f282e9/sass/lib/_reset.scss#L3